contracts/src/contracts/ramps/revolut/revolut.cairo

```diff
@@ -18,6 +21,14 @@ pub mod RevolutRamp {
     struct Storage {
         #[substorage(v0)]
         ownable: OwnableComponent::Storage,
+        token: ContractAddress,
+        // offchain_id -> address
+        offchain_registry: Map::<OffchainId, ContractAddress>,
+        next_liquidity_id: felt252,
+        // liquidity_id -> amount
+        liquidities: Map::<felt252, u256>,
+        // address -> liquidity_id
+        liquidity_by_address: Map::<ContractAddress, felt252>,
```

Comment:
The only useful storage here is `liquidities` that I think we could rename to `liquidity`